universal-login-relayer/src/integration/ethereum/WalletService.ts

```diff
@@ -31,6 +34,11 @@ class WalletService {
     this.hooks.emit('created', {transaction, contractAddress});
     return transaction;
   }
+
+  getTokensWithMinimalAmount(gasPrice: string) {
+    return this.supportedTokens.map((supportedToken) =>
+      ({...supportedToken, minimalAmount: safeMultiply(MINIMAL_DEPLOYMENT_GAS_LIMIT, gasPrice)}));
```

Comment:
Every token has gasPrice in own currency

Reply:
Yes, but currently we assume that for each token minimal amount is the same.